mindmeld/models/taggers/taggers.py

```diff
@@ -269,6 +269,7 @@ def get_entities_from_tags(query, tags):
         query (Query): Any query instance.
         tags (list of str): Joint app and system tags, like those
             created by get_tags_from_entities.
+        sys_resolver (SystemEntityRecognizer): Default to Duckling
```

Comment:
Can we name this `sys_entity_recognizer`? Right now its named `sys_resolver ` while the component is a recognizer.

Reply:
it was named `sy_resolver` since it was used for entity resolution, not recognition